context/app/static/js/components/detailPage/entityHeader/EntityHeaderContent/EntityHeaderContent.jsx

```diff
@@ -28,10 +27,10 @@ function EntityHeaderContent({ hubmap_id, entity_type, data, shouldDisplayHeader
       item && (
         <AnimatedFlexContainer style={props} key={key} maxWidth={vizIsFullscreen ? false : 'lg'}>
           {iconMap[entity_type]}
-          <EntityHeaderItem text={hubmap_id} />
-          {Object.entries(data).map(([k, v]) => (
-            <EntityHeaderItem text={v.value || `undefined ${v.label}`} key={k} />
-          ))}
+          {hubmap_id}
+          {Object.values(data)
+            .map((v) => v.value || `undefined ${v.label}`)
+            .join(' | ')}
```

Comment:
I understand moving from the divider to the pipe for the detail page `Summary` as it doesn't affect the page's appearance, but `EntityHeader` items are meant to be separated by lines which are the full height of its container. Also something has happened and the font size of the text/icons in the `EntityHeader` is different than before.

Reply:
Also missing a divider/pipe between the hubmap id and organ type.